Cargo.toml

```diff
@@ -58,18 +58,18 @@ repository = "uuid-rs/uuid"
 optional = true
 version = "0.2.0"
 
-[dependencies.md5]
+[dependencies.md-5]
 optional = true
-version = "0.7"
+version = "0.9.1"
```

Comment:
It could be worth to start with v0.8 versions of `md-5` and `sha-1` to avoid bumping MSRV. Updating versions can be done later in a separate PR.

Reply:
@newpavlov We've actually already bumped the MSRV so there's no reason not to track the latest versions here.